opentelemetry-sdk/src/export/mod.rs

```diff
@@ -8,4 +8,8 @@ pub mod logs;
 #[cfg_attr(docsrs, doc(cfg(feature = "trace")))]
 pub mod trace;
 
-pub use opentelemetry::ExportError;
+/// Marker trait for errors returned by exporters
```

Comment:
Not introduced in this PR but this is not a marker trait. Up to you if you want to fix this in this PR or later.